src/Jira/RsaSha1Signature.php

```diff
@@ -92,10 +92,10 @@ protected function baseString(Uri $url, $method = 'POST', array $parameters = []
     {
         $baseString = rawurlencode($method).'&';
         $schemeHostPath = $url->getScheme().'://'.$url->getHost();
-        if ($url->getPort() != '') {
```

Comment:
I've not read the code, but is it possible this is a change in behaviour (ie nulls will now meet the condition)?

Reply:
I've commited  ae15ea9abae633895bd23f183223be9376a4d92f to ensure that we compare with `null` (`getPort()` return an integer or `null`)